documentation/learnmore/existing.md

```diff
@@ -39,6 +89,21 @@ You can accomplish it like this:
 <pre class="console">&gt; flyway -baselineVersion="1" -baselineDescription="baseline_migration" baseline</pre>
 You must perform this step for each database that hasn't been cleaned by altering the [url](/documentation/configuration/parameters/url) again.
 
+### Example
+
+The Production instance needs to be baselined in our example, for which we can run:
+
+<pre class="console">&gt; flyway baseline -baselineVersion="1" -baselineDescription="baseline_migration" \
+  -url="jdbc:postgresql://instances.spawn.cc:&lt;ProdPort&gt;/pagila" \
+  -user="&lt;ProdUsername&gt;" -password="&lt;ProdPassword&gt;" -locations="filesystem:."</pre>
+
+This should give an output similar to:
+
+<pre class="console">Database: jdbc:postgresql://instances.spawn.cc:&lt;ProdPort&gt;/pagila (PostgreSQL 11.0)
+Creating Schema History table "public"."flyway_schema_history" with baseline ...
+1 rows affected
```

Comment:
We've made a change recently to hide `n rows affected` from regular output as people found it too spammy, so it'd make sense to delete it here too :)

Reply:
Done